File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution(object):
    def fizzBuzz(self, n):
        strr = []
        for i in range(1,n + 1):
            if i % 3 == 0 and i % 5 == 0:
                strr.append("FizzBuzz")
            elif i % 3 == 0:
                strr.append("Fizz")
            elif i % 5 == 0:
                strr.append("Buzz")
            else:
                strr.append(str(i))
        return strr

File: 0412-fizz-buzz/0412-fizz-buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let strr = [];
    
    for (let i = 1; i <= n; i++) {
        if (i % 3 === 0 && i % 5 === 0) {
            strr.push("FizzBuzz");
        } else if (i % 3 === 0) {
            strr.push("Fizz");
        } else if (i % 5 === 0) {
            strr.push("Buzz");
        } else {
            strr.push(i.toString());
        }
    }
    
    return strr;
};